### PR TITLE
added details for deleting security groups and DB subnet group

### PR DESCRIPTION
**Pull request by jtreuting**

### CODE REVIEW

File: deploy/Deprovisioning.AWS.md [hunk 1]
```diff
@@ -186,22 +186,65 @@ For more information see the AWS Documentation [Deleting Records](https://docs.a
 
 If the hosted zone was only being used for this particular tenant, you can delete the hosted zone as follows.
 
+> **NOTE:** You won't be able to delete the hosted zone if it contains anything other than an NS and an SOA record.
```
Comment (jtreuting) [inline deploy/Deprovisioning.AWS.md]: moved up for consistency across sub sections on placement

File: deploy/Deprovisioning.AWS.md [hunk 2]
```diff
@@ -186,22 +186,65 @@ For more information see the AWS Documentation [Deleting Records](https://docs.a
...
+
+##### AWS CLI
+
+The following code deletes a subnet group named `MySubnetGroup`
+
+`aws ec2 delete-db-subnet-group --db-subnet-group-name MySubnetGroup `
+
+**Delete the following subnet groups:**
+
+* [ ] DB subnet group: opus
```
Comment (malaffoon) [inline deploy/Deprovisioning.AWS.md]: Hmm. I've been reworking this document to make it more consistent with the other docs. I'll integrate these changes with mine. And then you can slam my review when i put it up.

Comment (jtreuting) [reply deploy/Deprovisioning.AWS.md]: looking forward to it! :)

**Comment by jtreuting** [comment]
closed since these changes were incorporated into a different PR that was merged already

**Pull request closed without merging**